test/config/configs.spec.js

```diff
@@ -135,6 +135,7 @@ describe('configs', function() {
     'no-propagate-all-parent-variables' : [ 'error', { version: '8.0' } ],
     'no-task-schedule': [ 'error', { version: '8.0' } ],
     'no-zeebe-properties': [ 'error', { version: '8.0' } ],
+    'secrets': [ 'warn', { version: '8.0' } ],
```

Comment:
From what I understand this will change for `8.3` and up only. Let's clarify this.

Reply:
This is based on https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/116#issuecomment-1678576712. Should we ask them again?

Reply:
8.0 as an answer makes no sense to me, as 8.0 ships with an entirely different connector run-time (that does not know the new pattern yet).